.github/workflows/build.yml

```diff
@@ -119,3 +120,119 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           file: ./lcov.info
+
+  package:
+    name: Package
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, macOS-latest, ubuntu-latest]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2019-11-04
+          override: true
+      - name: Install wasm-pack
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: wasm-pack    
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
```

Comment:
some steps have no nl, some have single nl, some have double nl here. It's not consistent.